hera_mc/sys_handling.py

```diff
@@ -282,19 +278,21 @@ def get_cminfo_correlator(self):
                 'cofa_lon': cofa_loc.lon,
                 'cofa_alt': cofa_loc.elevation}
 
-    def get_part_info(self, stn, at_date, part_name='post-amp'):
+    def get_part_from_type(self, stn, at_date, part_type='post-amp', include_ports=False):
```

Comment:
Why do we have the same method name here and in cm_hookup? That seems a little confusing.